### PR TITLE
Task/14 product value

### PR DESCRIPTION
**Pull request by brunobialas**
Task 14 done.

### CODE REVIEW

File: src/main/java/Model/Product.java [hunk 1]
```diff
@@ -2,13 +2,37 @@
...
-    this.value = value;
-    this.isOnSale = false;
+    this.regularValue = regularValue;
+    this.discountedValue = discountedValue;
+    this.isDiscounted = false;
+    }
+
+    public void discount(Integer percent) {
+    this.isDiscounted = true;
+    this.discountedValue = this.regularValue - (percent * this.regularValue);
```
Comment (jakubbialass) [inline src/main/java/Model/Product.java:19]: Percent is provided in the function as Integer, so percent will equal 20 or 50 or 10 etc.
It means that you can't calculate discountedValue as this.regularValue - (percent * this.regularValue).
You can do that for example like this: this.regularValue - (**percent/100** * this.regularValue).

Comment (jakubbialass) [inline src/main/java/Model/Product.java:19]: add tabulator

File: src/main/java/Model/Product.java [hunk 2]
```diff
@@ -2,13 +2,37 @@
...
+    }
+
+    public void removeDiscount() {
+    this.isDiscounted = false;
+    this.discountedValue = null;
```
Comment (jakubbialass) [inline src/main/java/Model/Product.java:24]: add tabulator

**Review by jakubbialass — CHANGES_REQUESTED** [review]

File: src/main/java/Model/Product.java (hunk 1 above)
Comment (brunobialas) [reply src/main/java/Model/Product.java:19]: Can I just change percent into Double or Float?

Comment (jakubbialass) [reply src/main/java/Model/Product.java:19]: No. Then the other programmer would need to check what that function actually does to be sure if he needs to pass 31.5 or 0.315.
Functions should be intuitive.

Comment (brunobialas) [reply src/main/java/Model/Product.java:19]: done

**Pull request merged**